Model/Sinch.php

```diff
@@ -3219,7 +3359,7 @@ private function addCategoryDataMultistore(
             );
 
             $this->_doQuery(
-                "INSERT IGNORE INTO $catalog_category_entity_varchar
+                "INSERT IGNORE INTO $catalog_category_entity_text
```

Comment:
This is more like it, but doesn't explain why in the other places, you've just added another query